rootfs/etc/nginx/lua/balancer.lua

```diff
@@ -150,15 +150,9 @@ local function sync_backends()
     return
   end
 
-  local backends_data = configuration.get_backends_data()
-  if not backends_data then
-    balancers = {}
-    return
-  end
-
-  local new_backends, err = cjson.decode(backends_data)
+  local new_backends = configuration.get_backends_data(is_init_call)
   if not new_backends then
-    ngx.log(ngx.ERR, "could not parse backends data: ", err)
```

Comment:
Why removing this log?

Reply:
Because the returned data is already parsed